features/admin.groups.v1/components/wizard/group-assign-users.tsx

```diff
@@ -239,16 +239,12 @@ export const AddGroupUsers: FunctionComponent<AddGroupUserProps> = (props: AddGr
         return;
     };
 
-    const handleAssignedItemCheckboxChange = (role: any) => {
-        const checkedGroups: any = [ ...checkedAssignedListItems ];
-
-        if (checkedGroups.includes(role)) {
-            checkedGroups.splice(checkedGroups.indexOf(role), 1);
-            setCheckedAssignedListItems(checkedGroups);
-        } else {
-            checkedGroups.push(role);
-            setCheckedAssignedListItems(checkedGroups);
-        }
+    const handleAssignedItemCheckboxChange = (role: UserBasicInterface) => {
```

Comment:
Let's add a doc comment to explain what this function does.

Reply:
I think the function naming itself explains the purpose.